src/js/chapter-factory.js

```diff
@@ -0,0 +1,87 @@
+import { makeDraggable, makeDroppable } from "./drag-drop";
+
+//Chapter factory
+const Chapter = (story, images, question, chapterNumber) => {
+  let completed = false;
+
+  const changeStatus = () => (completed = !completed);
+  const getStatus = () => completed;
+
+  return { story, images, question, chapterNumber, changeStatus, getStatus };
```

Comment:
Optional: Define getters not only for completed but for all properties? (Could be discussed/done also later)